connector_easypost/__openerp__.py

```diff
@@ -11,8 +11,11 @@
     'license': 'AGPL-3',
     'website': 'https://laslabs.com',
     'depends': [
+        'stock',
         'connector',
-        'stock_delivery_label_new',
+        'stock_picking_rate',
+        'stock_picking_package_info',
+        'base_delivery_carrier_label',
```

Comment:
I have updated connector in oca_dependancies.txt to explicitly require 9.0 for connector. The right branches are already provided for the repos providing ``stock_picking_rate``, ``stock_picking_package_info`` and ``base_delivery_carrier_label`` 😄